### DISCUSSION TITLE
Website Build

### DISCUSSION THREAD

**Discussion by MOHAMED-EHAB-DEV** [opensourcecommunity-hub/developer-portfolios#9] [General]
Hi,

Have u any plan to create a static website?
This website will preview all the templates.

**Comment by MOHAMED-EHAB-DEV** [comment]
@Susmita-Dey

**Reply by Susmita-Dey** [reply]
Yes, I do. First, we need a demo design a UI would be better. I can make it and then I'll get back to you.

**Reply by MOHAMED-EHAB-DEV** [reply]
> Yes, I do. First, we need a demo design a UI would be better. I can make it and then I'll get back to you.

I will create a new react app and design the page until you get back

**Reply by MOHAMED-EHAB-DEV** [reply]
> Yes, I do. First, we need a demo design a UI would be better. I can make it and then I'll get back to you.

First we need a repository for the code of the website

**Reply by Susmita-Dey** [reply]
> I will create a new react app and design the page until you get back

Okay, great.

**Reply by MOHAMED-EHAB-DEV** [reply]
> Yes, I do. First, we need a demo design a UI would be better. I can make it and then I'll get back to you.

Please make me a member of this organization

**Reply by MOHAMED-EHAB-DEV** [reply]
> > I will create a new react app and design the page until you get back
> 
> Okay, great.

I want to join
https://github.com/opensourcecommunity-hub/join-the-community/issues/41

**Reply by Susmita-Dey** [reply]
@Programming-School-Pro-Coding Use this repository itself. We'll use the data of the folks like their images and portfolio links present in this repo later on.

**Reply by MOHAMED-EHAB-DEV** [reply]
Ok, I will create A New Folder called website and put all the code in it

**Reply by Susmita-Dey** [reply]
> Ok, I will create A New Folder called website and put all the code in it

Ok

**Reply by MOHAMED-EHAB-DEV** [reply]
> > Ok, I will create A New Folder called website and put all the code in it
> 
> Ok

Hey Sir

There are some problems.

We Must add all the files in one file. Can I do this ?

The Portfolios folder is not in src folder in website folder

Can I move portfolios folder to src folder ?

**Reply by MOHAMED-EHAB-DEV** [reply]
@Susmita-Dey

**Reply by Susmita-Dey** [reply]
@Programming-School-Pro-Coding Yes you can move them. What do you mean by adding all the files into one?

**Reply by MOHAMED-EHAB-DEV** [reply]
> @Programming-School-Pro-Coding Yes you can move them. What do you mean by adding all the files into one?

@Susmita-Dey 
like this:
```json
{
    data: [
        {
            "portfolio_website": "https://www.susmitadey.com",
            "portfolio_code": "https://www.github.com/Susmita-Dey/portfolio",
            "portfolio_name": "Susmita Dey"
        }
    ]
}
```

**Reply by Susmita-Dey** [reply]
@Programming-School-Pro-Coding Okay, do it.

**Reply by MOHAMED-EHAB-DEV** [reply]
#11
